### PR TITLE
add linux-x86_64-centos6-epel-rdt image

### PR DESCRIPTION
**Pull request by jeroenooms**

### CODE REVIEW

File: public/data/platforms.json [hunk 1]
```diff
@@ -104,6 +104,19 @@
...
     {
+	"name": "linux-x86_64-centos6-epel-rdt",
+	"description": "CentOS 6 with Redhat Developer Toolset, R from EPEL",
+	"cran-name": null,
+	"rversion": "r-release",
+	"os-type": "Linux",
+	"cpu-type": "x86_64",
+	"os-info": "CentOS 6",
+	"compilers": "GCC 5.2.1",
+	"docker-image": "centos6-epel",
```
Comment (gaborcsardi) [inline public/data/platforms.json]: This should be `centos-epel-rdt` I believe.

File: public/data/platforms.json [hunk 2]
```diff
@@ -104,6 +104,19 @@
...
+	"sysreqs-platform": "linux-x86_64-centos6-epel-rdt"
```
Comment (gaborcsardi) [inline public/data/platforms.json]: And this should be `linux-x86_64-centos6-epel` I think. It must refer to a sysreqs platform from https://github.com/r-hub/sysreqsdb/tree/master/platforms.

Comment (gaborcsardi) [inline public/data/platforms.json]: I assume that this system is binary compatible with `linux-x86_64-centos6-epel`, right?

Comment (gaborcsardi) [inline public/data/platforms.json]: Or at least all entries here: https://github.com/r-hub/sysreqsdb/blob/master/platforms/linux-x86_64-centos6-epel.json apply to this platform, right?

Comment (jeroenooms) [inline public/data/platforms.json]: right that seems correct. The `-rdt` part only installs an additional c++ compiler and adds it to the PATH.

Comment (gaborcsardi) [inline public/data/platforms.json]: OK, thanks.

**Comment by gaborcsardi** [comment]
So can you pls add `-rdt` to the `docker-image`, and remove it from the `sysreqs-platform` field? Thanks.

**Comment by gaborcsardi** [comment]
Btw, i'll work out a way to make all this easier. We'll need to have a single central configuration database....

**Comment by jeroenooms** [comment]
OK updated.

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**